Formula/rbenv-bundler.rb

```diff
@@ -1,15 +1,12 @@
 class RbenvBundler < Formula
   desc "Makes shims aware of bundle install paths"
   homepage "https://github.com/carsomyr/rbenv-bundler"
-  url "https://github.com/carsomyr/rbenv-bundler/archive/1.00.tar.gz"
-  sha256 "84f5456b1ac8ea4554db8fa17c99b5f5d2eafcd647f85088a5d36e805c092ba7"
+  url "https://github.com/carsomyr/rbenv-bundler/archive/1.0.1.tar.gz"
+  sha256 "6840d4165242da4606cd246ee77d484a91ee926331c5a6f840847ce189f54d74"
   license "Apache-2.0"
-  revision 1
   head "https://github.com/carsomyr/rbenv-bundler.git"
 
-  bottle do
-    sha256 cellar: :any_skip_relocation, all: "d99c88c9492de9f01e61f3ba4d8930dbf1b607ac50c60d23c1e2e8963bd43419"
-  end
+  bottle :unneeded
```

Comment:
Why? Please don't edit the bottle block.

Reply:
Can you point out what's making the CI fail? Seems like it's bottle related.